templates/content/node--oe-publication--teaser.html.twig

```diff
@@ -4,12 +4,16 @@
  * Theme override to display a node of bundle publication in the teaser view mode.
  */
 #}
+{% set image_url = content.oe_publication_thumbnail|field_value|render %}
 {{ pattern('list_item', {
-  'variant': 'thumbnail_primary',
+  'variant': 'thumbnail_secondary',
   'url': url,
   'title': content.extra_field_oe_theme_helper_short_title_with_fallback|field_value,
   'meta': [
-    content.oe_publication_date|field_value
+    content.oe_publication_type|field_value|safe_join(', '),
```

Comment:
On the mockup pipe is used as separator between Resource type items.
<img width="896" alt="Screenshot 2020-11-27 at 12 39 54" src="https://user-images.githubusercontent.com/1183951/100434834-22219100-30ae-11eb-811e-839137bd2a58.png">

From spec - "This is the Primary Meta (fields are merged using "|")" - might be also it should be used for Author items too.

Reply:
@kugta  we should re-align requirements based on the above.